.build/main-app/app/scripts/built.js

```diff
@@ -1,56 +1,86 @@
-angular.module('tombola.snapCardGame.gameModel',[]);
+angular.module('tombola.snapCardGame.GameModel',[]);
```

Comment:
Nope - this is probably a service
Also the casing is incorrect for all of these: Tombola.SnapCardGame